src/xcode/ENA/ENA/Source/HTTPClient/__tests__/HTTPClient+DaysAndHoursTests.swift

```diff
@@ -143,8 +143,9 @@ final class HTTPClientDaysAndHoursTests: CWATestCase {
 			description: "expect error result"
 		)
 
-		let httpClient = HTTPClient.makeWith(mock: stack)
-		httpClient.fetchDay("2020-05-01", forCountry: "IT") { result in
+		let resource = FetchDayResource(day: "2020-05-01", country: "IT", signatureVerifier: MockVerifier())
```

Comment:
also here, should these tests not part of the fetch day resource so we can remove this file completely?

Reply:
Done